src/fidesops/ops/graph/config.py

```diff
@@ -306,7 +306,7 @@ class ObjectField(Field):
     fields: Dict[str, Field]
 
     @validator("data_categories")
-    def validate_data_categories(
+    def validate_data_categories(  # type: ignore
```

Comment:
Mypy error goes away if you remove the type for `cls`. `cls` is the class method version of `self` so mypy is fine with no annotation here. You may also need to add the `@classmethod` decorator, we have seen other errors when not using this.

```suggestion
    @validator("data_categories")
    @classmethod
    def validate_data_categories(
        cls, value: Optional[List[FidesOpsKey]]
    ) -> Optional[List[FidesOpsKey]]:
```